content/200-concepts/100-components/02-prisma-client/058-transactions.mdx

```diff
@@ -104,46 +102,70 @@ generator client {
 
 Then you can pass an async function into [`$transaction`](../../../guides/performance-and-optimization/prisma-client-transactions-guide#transaction-api).
 
-The following is an example of purchasing a movie ticket online. In this example, we really don't want to sell the same ticket twice:
+Let's say we're building an online banking system. One of the actions we want to perform is to send money from one person to another person.
+
+As experienced developers, we want to make sure that during the transfer,
+
+- the amount doesn't disappear
+- the amount isn't doubled
+
+This is a great use-case for interactive transactions because we need to perform logic in-between the writes to check the balance.
+
+In the example below, Alice and Bob each have $100 in their account. If you try to send more money than you have, the transfer is rejected.
+
+We'd expect Alice to be able to make 1 transfer for $100 and the other transfer would be rejected. This would result in Alice having $0 and Bob having $200.
```

Comment:
```suggestion
Alice is expected to be able to make 1 transfer for $100 while the other transfer would be rejected. This would result in Alice having $0 and Bob having $200.
```